### PR TITLE
LPS-184336 Create new columns in ObjectField entity | LPS-184877 Create Upgrade Process

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: 
https://issues.liferay.com/browse/LPS-184336;
https://issues.liferay.com/browse/LPS-184877.

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:object

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 1]
```diff
@@ -242,8 +242,8 @@ public ObjectField addOrUpdateSystemObjectField(
 			long userId, long objectDefinitionId, String businessType,
 			String dbColumnName, String dbTableName, String dbType,
 			boolean indexed, boolean indexedAsKeyword, String indexedLanguageId,
-			Map<Locale, String> labelMap, String name, String readOnly,
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java]: Maybe we will need this for the case of `objectDefinition.isUnmodifiableSystemObject()` (Bookmarks Object)

**Comment by guilhermedcamacho** [comment]
See #2170

**Pull request closed without merging**